Formula/apache-arrow.rb

```diff
@@ -16,31 +15,35 @@ class ApacheArrow < Formula
   depends_on "autoconf" => :build
   depends_on "cmake" => :build
   depends_on "boost"
+  depends_on "brotli"
+  depends_on "double-conversion"
   depends_on "flatbuffers"
+  depends_on "glog"
+  depends_on "grpc"
+  depends_on "llvm@7"
   depends_on "lz4"
   depends_on "numpy"
+  depends_on "openssl"
   depends_on "protobuf"
   depends_on "python"
+  depends_on "rapidjson"
+  depends_on "re2"
```

Comment:
@kou 
I've added some dependencies. 
https://github.com/apache/arrow/blob/master/cpp/Brewfile

Please let me know if my understanding is wrong.

Reply:
Additionally, we need to confirm that they are really used by `cmake` output.
It seems that we can't use build log on Jenkins: https://jenkins.brew.sh/job/Homebrew%20Core%20Pull%20Requests/44624/version=mojave/testReport/brew-test-bot/mojave/install___build_bottle_apache_arrow/
Because `cmake` output is truncated.

Could you confirm it locally?

Reply:
Thanks for your comments. 
I'll confirm it by running the following command, and remove unused libraries.
 `cmake -DCMAKE_BUILD_TYPE=release -DARROW_ORC=ON -DARROW_PARQUET=ON -DARROW_PLASMA=ON -DARROW_FLIGHT=ON -DARROW_PYTHON=ON -DDARROW_INSTALL_NAME_RPATH=OFF -DARROW_PROTOBUF_USE_SHARED=ON ..`

Reply:
`re2` is used by Gandiva.
Can we also enable Gandiva?

Reply:
Or should we work on this in another pull request?

Reply:
Yes. I'll try to enable Gandiva in another pull request.

Reply:
OK.